chain/index/msgindex.go

```diff
@@ -266,23 +218,6 @@ func PopulateAfterSnapshot(lctx context.Context, basePath string, cs ChainStore)
 	return nil
 }
 
-// init utilities
-func prepareDB(db *sql.DB) error {
-	for _, stmt := range dbDefs {
-		if _, err := db.Exec(stmt); err != nil {
-			return xerrors.Errorf("error executing sql statement '%s': %w", stmt, err)
-		}
-	}
-
-	for _, stmt := range dbPragmas {
```

Comment:
In lib/sqlite.go, we apply the pragma first during `Open` and then exec the DDL during `initDB`.
But here, we exec the DDL first and then apply the pragma. Are we sure that the creation of the pragma has not relevant effects to the execution of the DDL ?

Reply:
Yes, I did note this difference but judged it to be a mistake. The pragmas shouldn't impact creation _except_ for the page size which is set at db creation; so when these are first created they use the default page size, which has flow-on effects for the other settings that depend on page size.

The one where it matters now is `wal_autocheckpoint = 256` which we are setting - it's a multiple of page size, and the default page size is quite a bit lower than what we are setting, and the default `wal_autocheckpoint` is `1024`. But all this means is that we end up running checkpoints and possible compaction at a slightly smaller WAL size than before.

This may be a slight change of behaviour for users that have this feature turned on and then upgrade, but it shouldn't be noticeable. Also, message index is not on by default and it says this in the config file: `EXPERIMENTAL FEATURE. USE WITH CAUTION`, so I'm not sure who even is running it (I'm asking Glif and Zondax).